### PR TITLE
Product card changes

### PR DESCRIPTION
**Pull request by priyaljain01**

### CODE REVIEW

File: src/components/ProductCard/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      knitOrWoven: "Woven",
+      length: "Maxi",
+      mainTrend: "Bohemian",
+      neck: "V-Neck",
+      occasion: "Daily",
+      color: "Black",
+      style: "Dress",
+    },
+  },
+};
```
Comment (niteshMeena1) [inline src/components/ProductCard/index.stories.tsx:38]: Wrap this two object in enum then pass on story.

File: src/components/ProductCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      neck: string;
+      fabricType: string;
+      knitOrWoven: string;
+      mainTrend: string;
+      occasion: string;
+      color: string,
+      style: string,
+    };
+  };
+
```
Comment (niteshMeena1) [inline src/components/ProductCard/index.tsx:42]: no need to declare props value type declare only props type.

File: src/components/ProductCard/index.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+export const Default: Story = {
+  args: {
+    productCode: "00123456",
+    productDetail: {
+      productDetails: "Navy Blue printed woven empire %S, has a V-neck, three-quarter sleeves, and flared hem",
+      sizeAndFit: "Semi-Fit\nThe model (height 5'8\") is wearing a size S",
+      materialAndCare: "Cotton\nMachine-wash"
+    },
+    productSpecifications: {
```
Comment (deeporiontechnology) [inline src/components/ProductCard/index.stories.tsx:25]: create a data folder
and assign all these required objects in it
like productSpecifications, product details

So we can use them directly

File: src/components/ProductCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+const boxStyle = cva("flex flex-col gap-[8px]");
+
+function formatKey(key: string) {
+  return key
+    .replace(/([A-Z])/g, ' $1')
+    .replace(/^./, string => string.toUpperCase())
+    .trim();
+}
+
+type ProductCardProps = ComponentProps<"div"> &
```
Comment (deeporiontechnology) [inline src/components/ProductCard/index.tsx:21]: div ??
why you use div here
this is not correct

**Pull request merged**